board/safety/safety_toyota.h

```diff
@@ -94,7 +109,7 @@ static int toyota_tx_hook(CAN_FIFOMailBox_TypeDef *to_send) {
     if ((to_send->RIR>>21) == 0x343) {
       int desired_accel = ((to_send->RDLR & 0xFF) << 8) | ((to_send->RDLR >> 8) & 0xFF);
       desired_accel = to_signed(desired_accel, 16);
-      if (controls_allowed && toyota_actuation_limits) {
+      if (controls_allowed) {
```

Comment:
Why was the && here removed? Does toyota_actuation_limits toggle still work? If not, let's remove the safety mode entirely.

Reply:
We added it for convenience when testing, but I actually never used it. Yes, let me remove the safety mode.